torch/_meta_registrations.py

```diff
@@ -2695,191 +2696,527 @@ def meta_addbmm(self, batch1, batch2, *, beta=1, alpha=1):
+_foreach_unary_op(aten._foreach_exp.default, aten._foreach_exp_.default, torch.exp),
+_foreach_unary_op(aten._foreach_abs.default, aten._foreach_abs_.default, torch.abs),
+_foreach_unary_op(aten._foreach_acos.default, aten._foreach_acos_.default, torch.acos),
+_foreach_unary_op(aten._foreach_asin.default, aten._foreach_asin_.default, torch.asin),
+_foreach_unary_op(aten._foreach_atan.default, aten._foreach_atan_.default, torch.atan),
+_foreach_unary_op(aten._foreach_ceil.default, aten._foreach_ceil_.default, torch.ceil),
+_foreach_unary_op(aten._foreach_cos.default, aten._foreach_cos_.default, torch.cos),
+_foreach_unary_op(aten._foreach_cosh.default, aten._foreach_cosh_.default, torch.cosh),
+_foreach_unary_op(aten._foreach_erf.default, aten._foreach_erf_.default, torch.erf),
+_foreach_unary_op(aten._foreach_erfc.default, aten._foreach_erfc_.default, torch.erfc),
+_foreach_unary_op(
+    aten._foreach_expm1.default, aten._foreach_expm1_.default, torch.expm1
+),
+_foreach_unary_op(
+    aten._foreach_floor.default, aten._foreach_floor_.default, torch.floor
+),
+_foreach_unary_op(aten._foreach_log.default, aten._foreach_log_.default, torch.log),
+_foreach_unary_op(
+    aten._foreach_log10.default, aten._foreach_log10_.default, torch.log10
+),
+_foreach_unary_op(
+    aten._foreach_log1p.default, aten._foreach_log1p_.default, torch.log1p
+),
+_foreach_unary_op(aten._foreach_log2.default, aten._foreach_log2_.default, torch.log2),
+_foreach_unary_op(aten._foreach_neg.default, aten._foreach_neg_.default, torch.neg),
+_foreach_unary_op(aten._foreach_tan.default, aten._foreach_tan_.default, torch.tan),
+_foreach_unary_op(aten._foreach_tanh.default, aten._foreach_tanh_.default, torch.tanh),
+_foreach_unary_op(aten._foreach_sin.default, aten._foreach_sin_.default, torch.sin),
+_foreach_unary_op(aten._foreach_sinh.default, aten._foreach_sinh_.default, torch.sinh),
+_foreach_unary_op(
+    aten._foreach_round.default, aten._foreach_round_.default, torch.round
+),
+_foreach_unary_op(
+    aten._foreach_lgamma.default, aten._foreach_lgamma_.default, torch.lgamma
+),
+_foreach_unary_op(aten._foreach_frac.default, aten._foreach_frac_.default, torch.frac),
+_foreach_unary_op(
+    aten._foreach_trunc.default, aten._foreach_trunc_.default, torch.trunc
+),
+_foreach_unary_op(
+    aten._foreach_reciprocal.default,
+    aten._foreach_reciprocal_.default,
+    torch.reciprocal,
+),
+_foreach_unary_op(
+    aten._foreach_sigmoid.default, aten._foreach_sigmoid_.default, torch.sigmoid
+),
+_foreach_unary_op(aten._foreach_sign.default, aten._foreach_sign_.default, torch.sign),
+_foreach_unary_op(aten._foreach_zero.default, aten._foreach_zero_.default, torch.zero),
+
+
+_foreach_pointwise_op_scalar(
+    aten._foreach_addcdiv.Scalar, aten._foreach_addcdiv_.Scalar, torch.addcdiv
+)
+# _foreach_pointwise_op_scalar(aten._foreach_addcmul.Scalar, aten._foreach_addcmul_.Scalar, torch.addcmul)
```

Comment:
addcmul definitions here should work as soon as core and ref match when dealing with a float scalar and int tensors (mentioned above).